### ISSUE TITLE
pip struggles to install ASPIRED

### ISSUE THREAD

**Issue by cylammarco** [cylammarco/ASPIRED#6]
Reported from Ilknur:

`pip install git+https://github.com/cyalmmarco/ASPIRED.git@dev` installs the dist-info but not the package

however
`git clone https://github.com/cyalmmarco/ASPIRED.git@dev` then `pip install -e ASPIRED` works

Is it a problem with pip or with setup.py? Probably the latter?

**Comment by jveitchmichaelis** [comment]
Have you tried cleaning first? Delete all the build folders etc

**Comment by cylammarco** [comment]
Should also sort out plotly-orca dependency issue here. It should be an optional dependency.

**Comment by cylammarco** [comment]
SpectRes v2.1.1 fixed the wavelength coverage problem, it is now included in the setup.py which installs the package from pypi. Waiting on RASCAL release before looking into this again.

**Comment by cylammarco** [comment]
Dependency handling should be simpler because of the recent updates with SpectRes and RASCAL:

[SpectRes](https://github.com/ACCarnall/SpectRes) v2.1.1 has fixed the last pixel bug and can handle new wavelength range outside of the old wavelength range, dependency on a forked version is no longer needed.

[RASCAL](https://github.com/jveitchmichaelis/rascal) master branch is usable now.

I will do more build test before closing.

**Comment by cylammarco** [comment]
Added MANIFEST.in, which solved some of the problem.
